packages/components/src/logos/big-sky-logo/index.tsx

```diff
@@ -0,0 +1,49 @@
+interface BigSkyLogoProps extends Omit< React.SVGProps< SVGSVGElement >, 'height' | 'width' > {
+	height?: number;
+	monochrome?: boolean;
+	title?: string;
+	normalized?: boolean;
+}
+
+const Mark = ( {
```

Comment:
Thank you for following the logo specs that I've been working on !

I'll follow-up soon with a few tweaks:

- make the `normalized` version the default version
- consider  the `normalized` prop for now (we'll add it to the logos if needed)
- if it needs to be kept around, consider tweaking the `viewbox` instead of scaling the logo
- add `aria-labelledby` to the `svg`
- tweak the default `title` by removing "Logo"
- consider marking the title for translation (should we? @aduth @mirka )

Reply:
I think we'd probably be fine with keeping product names like this untranslated.

Reply:
Follow-up PR: #103612